packages/fluentui/docs/src/components/Sidebar/Sidebar.tsx

```diff
@@ -561,6 +566,11 @@ const Sidebar: React.FC<RouteComponentProps & SidebarProps> = props => {
           setActiveItemIds(activeItemIds);
         }}
       />
+      <Flex column>
```

Comment:
As we have just one example and 0 results at the moment, I would not add the item to sidebar.

Comment this out, add `//TODO: enable after Nov 2020 when we have data`